BD-Automation-Engine/REQUIRED_EXPORTS.md

```
@@ -0,0 +1,291 @@
+3. Configuration values
+4. Architecture patterns
+5. Next steps discussed
+```
+
+---
+
+## File Placement Summary
+
+After all exports, your folder structure should look like:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)